src/Commands/Workflow/Info/LogsCommand.php

```diff
@@ -18,9 +18,9 @@ class LogsCommand extends InfoBaseCommand
      * @param string $site_id Name or UUID of the site that the workflow belongs to
      * @option string $id The UUID of a specific workflow to show
      *
-     * @usage terminus workflow:info:logs <site_name> --id=<workflow>
```

Comment:
I think you've based off an old branch. I'm _very_ sure Ben fixed these. They should use `<site>`.